test/CMakeLists.txt

```diff
@@ -175,4 +189,6 @@ endif()
 add_subdirectory(atm)
 
 # gdas aerosol tests
-add_subdirectory(aero)
+if (${BUILD_GDASBUNDLE})
```

Comment:
why this change in this PR?

Reply:
If it's not needed, I can remove it.

Reply:
It's mainly that if you aren't doing anything to `test/aero` then we shouldn't change that `add_subdirectory` part right now

Reply:
Yeah I would revert this change in this PR, just so that we only touch it later when we add 3dvar or hofx tests

Reply:
done.